projects/ad7768-1fmcz/src/ad77681evb.c

```diff
@@ -1,5 +1,5 @@
 /***************************************************************************//**
-* @file ad77681_evb.c
+* @file ad77681evb.c
```

Comment:
nitpick: This is related with the previous commit

Reply:
It seems to me that this is the name of the file, otherwise doxygen will complain.

Reply:
I thought you have renamed it, it is ok then.